src/editor/build/build_manager.h

```diff
@@ -0,0 +1,45 @@
+
+#ifdef TOOLS_ENABLED
+
+#ifndef GODOT_JVM_BUILD_MANAGER_H
+#define GODOT_JVM_BUILD_MANAGER_H
+
+#include <core/error_list.h>
+#include <core/os/os.h>
+#include <core/os/thread.h>
+
+class BuildManager {
+public:
```

Comment:
It misses deletion of assignement operator and copy constructor on this meyer singleton.